src/bin/zmcontrol

```diff
@@ -86,9 +84,6 @@ my %allservices = (
     "mta"              => "/opt/zextras/bin/zmmtactl",
     "mailbox"          => "/opt/zextras/bin/zmstorectl",
     "service"          => "/opt/zextras/bin/zmstorectl",
-    "zimbra"           => "/opt/zextras/bin/zmstorectl",
```

Comment:
Do we still need to remove these entries, their start/stop order?
Please check removals from line 56-58, 75-77 and this.